styles/index.scss

```diff
@@ -241,25 +242,27 @@ ol {
   position: relative;
   display: block;
   width: 24px;
-}
 
-.close-button:before {
-  content: '';
-  display: block;
-  position: absolute;
-  border: 1px solid colors.$color-white;
-  border-radius: 2px;
-  transform: rotate(45deg);
-  width: 24px;
+  &:before {
+    content: '';
+    display: block;
+    position: absolute;
+    border: 1px solid colors.$color-white;
+    border-radius: 2px;
+    transform: rotate(45deg);
+    width: 24px;
+  }
 }
 
-.close-button:after {
-  content: '';
-  display: block;
-  position: absolute;
-  border: 1px solid colors.$color-white;
-  border-radius: 2px;
-  transform: rotate(-45deg);
-  left: -1px;
-  width: 24px;
+.close-button {
+  &:after {
```

Comment:
can we move this `&:after` into the existing `.close-button` styles above like you did with the `:before`?